src/net.cpp

```diff
@@ -751,16 +751,16 @@ const uint256& V1TransportDeserializer::GetMessageHash() const
     return data_hash;
 }
 
-Optional<CNetMessage> V1TransportDeserializer::GetMessage(const std::chrono::microseconds time, uint32_t& out_err_raw_size)
+CNetMessage V1TransportDeserializer::GetMessage(const std::chrono::microseconds time, bool& reject_message)
 {
     // decompose a single CNetMessage from the TransportDeserializer
```

Comment:
In commit "Remove Optional from GetMessage" (eb21969fe3fcd845d4060d890f7c46d703238661)

Might be good to initialize `reject_message = false` here just to be clear this is an output parameter, not an in/out parameter

Reply:
Good idea, thanks!

(Maybe one day C++ will have the in/out/ref labels from C#)

Reply:
I hesitated away from this just in case the caller wanted to reject the message for some other reason before calling GetMessage.  (So reject_message would already be true)

However, now writing that out into words, I think that's an unlikely scenario, and I agree that setting it at the beginning makes things clearer.

Thanks, made the change!